pythainlp/transliterate/thai2rom.py

```diff
@@ -3,9 +3,6 @@
 Romanization of Thai words based on machine-learnt engine ("thai2rom")
 """
 
-import random
-
-import numpy as np
```

Comment:
Original model need to be removed `numpy` from `forward()` for ONNX conversion to work properly